client_code/ProgressBar/__init__.py

```diff
@@ -8,24 +8,14 @@
 
 css = """ .anvil-role-ae-progress-track, .anvil-role-ae-progress-indicator {
     display: block;
-    height: 3px;
     margin: 0;
 }
 
 .anvil-role-ae-progress-track {
     width: 100%;
 }
 
-.anvil-role-ae-progress-indicator {
-    top: 0 !important;
-}
-
-.anvil-role-ae-progress-track > .holder, .anvil-role-ae-progress-indicator > .holder {
-    display: block !important;
-}
-
```

Comment:
Do we know why we're removing this? Doesn't seem related to the PR?

Reply:
When I just tried setting height properties with it left it and it was making the code not work or some awkward height settings.